electron/install.js

```diff
@@ -7,6 +7,11 @@ const process = require('process');
 const { spawnSync } = require('child_process');
 const Docker = require('dockerode');
 
+/**
+ * current version of the pearl release
+ * - use "" (nothing as a suffix) for latest release candidate, for example "0.1.0rc26"
+ * - use "alpha" for alpha release, for example "0.1.0rc26-alpha"
+ */
 const Version = '0.1.0rc26';
```

Comment:
not your change but could you refactor this to: `const OlasMiddlewareVersion`
should avoid confusion for others in future